test/hotspot/jtreg/compiler/c2/irTests/TestIfMinMax.java

```diff
@@ -33,7 +33,6 @@
  * @bug 8324655 8329797
  * @key randomness
  * @summary Test that if expressions are properly folded into min/max nodes
- * @requires os.arch != "riscv64"
```

Comment:
This is just a drive-by but I think this change reverts [JDK-8329355](https://bugs.openjdk.org/browse/JDK-8329355). It might have been accidentally removed in this merge: https://github.com/openjdk/jdk/commit/e30fa929b19e8b7d6db9073fdf640938d3db39fd

Reply:
Good catch. I sync'd this file back up with `openjdk/jdk:master`.